airdrop-cli/src/index.ts

```diff
@@ -6,6 +6,6 @@ const commandsPath = path.join(__dirname, "commands");
 const cli = new CLI("npm run", commandsPath);
 
 const shim = new Shim(cli);
-shim.execute(process.argv);
+shim.execute(process.argv).catch(console.error);
```

Comment:
If you have the latest linter settings from ts-template then it should warn you of any promises that aren't caught. 

Remember to run `yarn format`!

Reply:
If I remember right I'm sure that's why I caught it, trying to get into the habit